funnel/templates/session_view_popup.html.jinja2

```diff
@@ -18,7 +18,7 @@
       </p>
     {% endif %}
     {% if session.venue_room %}
-      <p class="mui--text-subhead zero-bottom-margin"><small class="mui--text-light">in</small> {{ session.venue_room.title }} <small class="mui--text-light">{{ session.venue_room.venue.title }}</small></p>
+      <p class="mui--text-subhead zero-bottom-margin" data-cy-session="room"><small class="mui--text-light">in {{ session.venue_room.venue.title }} – {{ session.venue_room.title }}</small></p>
```

Comment:
UI change, should be separate from this PR.

Reply:
Changed it back